src/components/CardList/CardList.jsx

```diff
@@ -0,0 +1,15 @@
+import cardData from 'src/data/products.js';
+import Card from 'src/components/Card/Card.jsx';
+import styles from './CardList.module.css';
+
+const CardList = () => {
+    return (
+        <div className={styles.cardList}>
+            {cardData.map((item) => (
```

Comment:
I'm not sure it's a good idea import data for the list here as in the future you will need to render them depends on selected category

Reply:
Agree, I was planning to adjust the code in the future when the task will require it. But I see the point! I changed the code  as you suggest.